src/Benchmarks/DeserializationBenchmark.cs

```diff
@@ -28,18 +28,18 @@ public void BenchmarkSetup()
             var xmlFileName = Path.Combine("TestData", "fp-test-patient.xml");
             XmlData = File.ReadAllText(xmlFileName);
 
-            XmlDeserializer = new BaseFhirXmlPocoDeserializer(typeof(TestPatient).Assembly);
-            JsonDeserializer = new BaseFhirJsonPocoDeserializer(typeof(TestPatient).Assembly);
+            XmlDeserializer = new FhirXmlPocoDeserializer();
+            JsonDeserializer = new FhirJsonPocoDeserializer();
```

Comment:
If we're changing this anyways, we may as well use an IFhirSerializationEngine here